docs/installation/windows.md

```diff
@@ -321,7 +322,7 @@ installer](https://www.docker.com/toolbox).
 
 ## Container port redirection
 
-If you are curious, the username for the Docker default user is `docker` and the
+If you are curious, the username for the Docker default VM user is `docker` and the
```

Comment:
nit; I think we can;

s/Docker default VM user/Docker default VM/

(remove "user"), because we're already talking about "username" and "password"